reana-ui/src/pages/workflowList/components/WorkflowStatusFilter.js

```diff
@@ -2,39 +2,94 @@
   -*- coding: utf-8 -*-
 
   This file is part of REANA.
-  Copyright (C) 2020 CERN.
+  Copyright (C) 2020, 2022 CERN.
 
   REANA is free software; you can redistribute it and/or modify it
   under the terms of the MIT License; see LICENSE file for more details.
 */
 
 import PropTypes from "prop-types";
-import { Dropdown } from "semantic-ui-react";
+import { useEffect, useState } from "react";
+import { Checkbox, Dropdown, Grid } from "semantic-ui-react";
 
+import { WORKFLOW_STATUSES } from "~/config";
 import { statusMapping } from "~/util";
 
+function removeDeleted(statuses) {
+  return statuses.filter((status) => status !== "deleted");
+}
+
+const NON_DELETED_STATUSES = removeDeleted(WORKFLOW_STATUSES);
+
 const statusOptions = Object.keys(statusMapping).map((status) => ({
   key: status,
   text: status,
   value: status,
   icon: statusMapping[status].icon,
 }));
 
-export default function WorkflowStatusFilters({ valueList, filter }) {
+const nonDeletedStatusOptions = statusOptions.filter(({ value: status }) =>
+  NON_DELETED_STATUSES.includes(status)
+);
```

Comment:
Very minor suggestion: we could avoid defining `WORKFLOW_STATUSES` (even though it's perfectly fine to keep it, since it might be useful in the future!) by doing this:
```js
const nonDeletedStatusOptions = statusOptions.filter(
  (status) => status.value !== "deleted"
);

const nonDeletedStatuses = nonDeletedStatusOptions.map(
  (status) => status.key
);
```

Reply:
I think we can keep this, because as you say it will probably be useful in the future